File: LeetCode/0217. Contains Duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int, int> umap;
        for (int i = 0; i < nums.size(); i++) {
            umap[nums[i]]++;
        }
        for (auto it : umap) {
            if(it.second >= 2)
                return true;
        }
        return false;
    }
};

File: LeetCode/0217. Contains Duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        hashset = set()

        for num in nums:
            if num in hashset:
                return True
            hashset.add(num)
        return False
